### PR TITLE
OSD: Display FPS/VPS when "Show FPS" is enabled

### PR DESCRIPTION
**Pull request by stenzek**
This branch changes two things:

 - Displays VPS as well as FPS in the OSD overlay when the "show fps" option is enabled. FPS on its own can be decoupled from the display refresh rate in variable framerate games, and does not differentiate between the emulator running slow, or game not rendering 30/50/60FPS.
-  Removes the "FPSCounter" class. This was a duplicate source of information as Core tracks it already.

I'm tagging this as RFC because I'm not sure whether including the speed percentage in the overlay as well is a worthwhile idea. Knowing the region of the game and the VPS is enough to tell whether the game is running full-speed or not, and I didn't want to bloat the OSD with any more information. However, it may be useful to Android users who do not see this information in the title bar (as there is none).

There is still the issue of what we actually define "FPS" as, as well. At the moment, it's "unique XFB copies scanned out per second" without immediate XFB enabled, or the number of XFB copies with immediate enabled. This is not indicative of the game's internal performance for the former case, but I wanted to keep these changes separate.

### CODE REVIEW

File: Source/Core/VideoCommon/VideoCommon.vcxproj.filters [hunk 1]
```diff
@@ -176,9 +173,6 @@
     <ClCompile Include="AbstractTexture.cpp">
       <Filter>Base</Filter>
     </ClCompile>
-    <ClCompile Include="AbstractRawTexture.cpp">
```
Comment (lioncash) [inline Source/Core/VideoCommon/VideoCommon.vcxproj.filters]: was it intentional for this to be removed (along with `TextureConvertionShaderGen` being corrected). Seems like something that should be in a separate PR.

Comment (stenzek) [reply Source/Core/VideoCommon/VideoCommon.vcxproj.filters]: Hmm, yep, looks like VS auto-removed it since the file no longer exists. I'll make a separate PR.

**Comment by degasus** [comment]
I agree, the current way to generate the FPS twice in an incompatible way is stupid. So this sounds like a good step forward.

File: Source/Core/VideoCommon/RenderBase.cpp [hunk 2]
```diff
@@ -255,7 +254,10 @@ void Renderer::DrawDebugText()
   if (g_ActiveConfig.bShowFPS || SConfig::GetInstance().m_ShowFrameCount)
   {
     if (g_ActiveConfig.bShowFPS)
-      final_cyan += StringFromFormat("FPS: %.2f", m_fps_counter.GetFPS());
+    {
+      const Core::PerformanceStatistics& pstats = Core::GetPerformanceStatistics();
```
Comment (degasus) [inline Source/Core/VideoCommon/RenderBase.cpp]: Isn't this a race condition?
